.eslintrc.json

```diff
@@ -12,9 +14,7 @@
       "json"
     ],
     "rules": {
-        "indent": ["error",
-            2
-        ],
+        "indent": ["error", 2],
```

Comment:
one-lined all these so it'd be easier to sort rules A-Z. I had a real heck of a time trying to quickly find rules in the unsorted list. But sorting everything in this PR got a bit too noisy.